packages/fscomponents/src/components/AddressForm.tsx

```diff
@@ -33,171 +33,163 @@ const S = StyleSheet.create({
   }
 });
 
-export class AddressForm extends Component<AddressFormProps> {
-  form: any;
-  fieldsStyleConfig: any;
-  fieldsTypes: any;
-  fieldsOptions: any;
-  labelPosition: FormLabelPosition;
 
-  constructor(props: AddressFormProps) {
-    super(props);
+export const AddressForm = (props: AddressFormProps): JSX.Element => {
```

Comment:
Please add the React.FunctionComponent typing.

Reply:
fixed